client/customer-effort-score-tracks/customer-effort-score-tracks.js

```diff
@@ -98,10 +98,11 @@ function CustomerEffortScoreTracks( {
 		} );
 	};
 
-	const recordScore = ( score ) => {
+	const recordScore = ( score, comments ) => {
 		recordEvent( 'ces_feedback', {
 			action,
 			score,
+			comments,
```

Comment:
If comments are not entered, they get sent as `undefined`... do we know if this will cause any issues actually recording to Tracks? I don't *think* so.

Reply:
I don't think so either, but I've changed this to use `''` if `comments` is falsey. Thanks :-)